src/assets/scss/application.scss

```diff
@@ -290,4 +290,38 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  float: right;
+  border-bottom: 1px solid $govuk-border-colour;
```

Comment:
The component in the design which is linked to in the JIRA ticket does not have a border on the bottom. Obviously you have added this deliberately but as there is a discrepancy I just wanted to make sure which design is the correct one: border or no border? 🤔

Reply:
It was found at a later stage (UCD huddle) that the language component should have a bottom border, that's why it was added.